src/common/GadgetDescription/index.tsx

```diff
@@ -53,7 +53,12 @@ export function GadgetDescription({
     const allInstances = JSON.parse(localStorage.getItem('headlamp_embeded_resources') || '[]');
     const instance = allInstances.find(instance => instance.id === id);
     if (instance) {
-      setEnableHistoricalData(!!instance.isHeadless);
+      // this would happen when you come to details page for the first time
+      if (instance.isHeadless === undefined) {
+        setEnableHistoricalData(true);
+      } else {
+        setEnableHistoricalData(!!instance.isHeadless);
+      }
```

Comment:
Is this the same as:
`setEnableHistoricalData(instance.isHeadless ?? true)`

Reply:
updated